client/my-sites/checkout/checkout/style.scss

```diff
@@ -1187,118 +1187,3 @@
 		height: 40px;
 	}
 }
-
-/* Payment Methods Test
-======================================= */
-.payment-box__pm-test-wrapper {
-	background: var( --color-surface );
-	padding: 10px 8px 0;
-
-	@include breakpoint( '>660px' ) {
-		padding: 25px 30px 0;
-	}
-}
-
-.payment-box__pm-title {
-	font-size: 16px;
-	margin-bottom: 8px;
-}
-
-.payment-box__pm-wrapper {
-	margin: 0;
-	padding: 0;
-
-	.section-nav-tab {
-		list-style: none;
-		border: 0;
-		margin: 8px 0 0;
-		position: relative;
-		z-index: z-index( 'root', '.payment-box__pm-wrapper .section-nav-tab' );
-
-		&:first-child {
-			margin-top: 0 !important;
-		}
-
-		&::after {
-			display: block;
-			content: '';
-			position: absolute;
-			top: 50%;
-			transform: translateY( -50% );
-			left: 15px;
-			width: 21px;
-			height: 21px;
-			border: 1px solid var( --color-neutral-10 );
-			border-radius: 50%;
-			box-sizing: border-box;
-			z-index: z-index( 'root', '.payment-box__pm-wrapper .section-nav-tab::after' );
```

Comment:
And here! https://github.com/Automattic/wp-calypso/blob/4db3529a9bb9c6869e6587dc91a3acbf1ab732eb/client/assets/stylesheets/shared/functions/_z-index.scss#L38

Reply:
Thanks for the catch. I'll take it out.